File: output/ChatGPT/avatar/Python/Java/atcoder_ABC154_A.java
public class atcoder_ABC154_A {
    public static void main(String[] args) {
        solve();
    }

    public static void solve() {
        Scanner in = new Scanner(System.in);
        String S = in.next();
        String T = in.next();
        int A = in.nextInt();
        int B = in.nextInt();
        String U = in.next();

        if (S.equals(U)) {
            A -= 1;
        }
        if (T.equals(U)) {
            B -= 1;
        }

        System.out.println(A + " " + B);
    }
}

File: output/ChatGPT/avatar/Python/C++/atcoder_ABC154_A.cpp
#include <iostream>
#include <string>

void solve(std::string S, std::string T, int A, int B, std::string U) {
    if (S == U) {
        A -= 1;
    }
    if (T == U) {
        B -= 1;
    }
    std::cout << A << " " << B << std::endl;
}

int main() {
    std::string S, T, U;
    int A, B;
    
    std::cin >> S >> T >> A >> B >> U;
    solve(S, T, A, B, U);
    
    return 0;
}